docpages/advanced_reference/separate-events.md

```diff
@@ -16,54 +16,49 @@ Once that's done, it should look similar to this (this screenshot has more files
 
 First, we need to define the function that will be called when the event fires. We do this in the `message_listener.h`, like so:
 
-~~~~~~~~~~{.cpp}
+~~~~~~~~~~cpp
 #pragma once
 
 #include <dpp/dpp.h>
 
 class message_listener {
-
 public:
-
-    /* Create a static function that can be called anywhere. */
-    static void on_message_create(const dpp::message_create_t& event);
-
+	/* Create a static function that can be called anywhere. */
+	static void on_message_create(const dpp::message_create_t& event);
 };
 ~~~~~~~~~~
 
 Then we need to add our code for what should happen when this event fires. We do this in the `message_listener.cpp`, like so:
 
-~~~~~~~~~~{.cpp}
+~~~~~~~~~~cpp
 #include "message_listener.h"
 
 void message_listener::on_message_create(const dpp::message_create_t &event) {
-    /* See if the message contains the phrase we want to check for.
-     * If there's at least a single match, we reply and say it's not allowed.
-     */
-    if (event.msg.content.find("bad word") != std::string::npos) {
-        event.reply("That is not allowed here. Please, mind your language!", true);
-    }
+	/* See if the message contains the phrase we want to check for.
```

Comment:
What's happened to the Tab indents here? They've doubled

Reply:
its because its converted to tabs, and github displays tabs as different indent amounts

Reply:
> its because its converted to tabs, and github displays tabs as different indent amounts

ahhhhh